src/agentscope/tts/_tts_base.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+"""The TTS model base class."""
+
+from abc import ABC, abstractmethod
+from typing import Any
+
+from agentscope.message import Msg
+
+from ._tts_response import TTSResponse
+
+
+class TTSModelBase(ABC):
+    """Base class for TTS models."""
+
+    # Class attribute to indicate if this TTS model supports streaming input
+    supports_streaming_input: bool = False
+
+    def __init__(
+        self,
+        model_name: str,
+        stream: bool = True,
```

Comment:
Since some TTS models don't support real-time generation, why include the `stream` argument in the base class?